### PR TITLE
Docs/eu update endpoints

### LINKED ISSUE #82: Update README with corrected endpoints

### PR DESCRIPTION
**Pull request by ErinUntermeyer**
#### Type of Change Made 
- [ ] bugfix 
- [x] new feature 
- [ ] breaking change 
- [ ] testing 
- [ ] formatting 
- [ ] styling
#### What's this PR do?
- Updated the README with new endpoints.
#### Where should the reviewer start?
- README.md
#### What are the relevant tickets?
- Closes #82

### CODE REVIEW

**Review by amykarnaze — APPROVED** [review]
These responses will complement the changes we made to ensure better typing on the front end.

**Review by GreysonElkins — APPROVED** [review]
This all looks good to me!

**Comment by gabichuelas** [comment]
Did any of the other responses change beyond the ones for all applicants / single applicant? Looks like all that's being added is extra indentation.. want to make sure im not missing anything. @rrabinovitch  ?

File: README.md [hunk 1]
```diff
@@ -81,115 +81,149 @@ This repo is the back-end service for HireUp and is consumed by our front-end ap
...
-      "first_name": "Greyson",
-      "last_name": "Johns",
-      "bio": "I'm the best one you could possibly hire",
-      "username": "Chipmunk",
-      "skills": [2, 3],
-      "values": [2]
+		"first_name": "Greyson",
+		"last_name": "Johns",
+		"bio": "I'm the best one you could possibly hire",
+		"username": "Chipmunk",
```
Comment (rrabinovitch) [inline README.md:171]: just realized we're missing email here! I'll add it in my PR

**Review by rrabinovitch — APPROVED** [review]

**Pull request merged**